### PR TITLE
Validation text fix

### PR DESCRIPTION
**Pull request by irubido**

### CODE REVIEW

File: src/renderer/containers/Onboard/SigningKey/Import/validation.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {StringSchema} from "@hapi/joi";
 
 export const PRIVATE_KEY_NOT_STRING_MESSAGE="Value must be non empty string";
 export const PRIVATE_KEY_WRONG_CHARACTERS_MESSAGE = "Private key must contain hex characters only";
-export const PRIVATE_KEY_WRONG_LENGTH_MESSAGE = `Private key should have ${SECRET_KEY_LENGTH} bytes`;
+export const PRIVATE_KEY_WRONG_LENGTH_MESSAGE = `Private key should have ${SECRET_KEY_LENGTH} characters`;
```
Comment (mpetrunic) [inline src/renderer/containers/Onboard/SigningKey/Import/validation.ts]: SECRET_KEY_LENGTH*2 characters

each byte is represented as 2 hex characters

**Comment by mpetrunic** [comment]
I think we have same error for public key

File: src/renderer/containers/Onboard/SigningKey/Import/validation.ts (hunk 1 above)
Comment (irubido) [reply src/renderer/containers/Onboard/SigningKey/Import/validation.ts]: SECRET_KEY_LENGTH constant was set to 64, so i only changed text

"export declare const SECRET_KEY_LENGTH = 64;"

I'll change constant to 32 then and set {SECRET_KEY_LENGTH*2} characters

Comment (mpetrunic) [reply src/renderer/containers/Onboard/SigningKey/Import/validation.ts]: Hmm that's odd, can you check where else this constant is used?

If nowhere, set it to 32 and multiply by two here

Comment (irubido) [reply src/renderer/containers/Onboard/SigningKey/Import/validation.ts]: I changed it and committed changes, but it does not show on pull request, strange, and I couldn't see it git either. I checked, its only used for this case.

Comment (mpetrunic) [reply src/renderer/containers/Onboard/SigningKey/Import/validation.ts]: I think we should still set to 32 as this is missleading

**Pull request merged**